extension/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQListenerTests.cs

```diff
@@ -43,12 +43,12 @@ public RabbitMQListenerTests()
         [Fact]
         public void CreatesHeadersAndRepublishes()
         {
-            var queueName = "blah";
+            string queueName = "blah";
```

Comment:
In reference to the previous question, is there a reason to remove `var` here?

Reply:
The `var`s were not uniformly applied in the source code when literal values were being assigned to variable. We had to choose between var or explicit types. The style rules are set such that only when the type-name is repeated on both left and right side of the assignment operator, we will use var otherwise won't.

Reply:
Thanks for the clarification.